File: PBS/week-6/sentimental-mario-less/mario.py
import cs50
import sys

height = cs50.get_int("Height: ")

while height <= 0 or height > 8:
    height = cs50.get_int("Height: ")

for i in range(height):
    for s in range(height - i - 1):
        sys.stdout.write(" ")
        sys.stdout.flush()
    for j in range(i + 1):
        sys.stdout.write("#")
        sys.stdout.flush()
    print("")

File: PBS/week-1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;

    do
    {
        height = get_int("Height: ");
    }
    while (height <= 0 || height > 8);

    for (int i = 0; i < height; i++)
    {
        for (int s = 1; s < height - i; s++)
        {
            printf(" ");
        }
        for (int c = 0; c < 2; c++)
        {
            for (int j = 0; j <= i; j++)
            {
                printf("#");
            }
            if (c == 0)
            {
                printf("  ");
            }
        }

        printf("\n");
    }
}